content/tutorials/manage-account-settings/set-up-multi-signing.md

```diff
@@ -16,7 +16,7 @@ This tutorial demonstrates how to enable multi-signing for an address.
 
 ## 1. Prepare a funded address
 
-You need an XRP Ledger address that can send transactions, and has enough XRP available. Multi-signing requires more than the usual amount of XRP for the [account reserve](reserves.html) and [transaction cost](transaction-cost.html), increasing with the number of signers and signatures you use.
+You need an XRP Ledger address that can send transactions, and has enough XRP available. Without the [MultiSignReserve Amendment](known-amendments.html#multisignreserve), multi-signing requires more than the usual amount of XRP for the [account reserve](reserves.html) and [transaction cost](transaction-cost.html), increasing with the number of signers and signatures you use. With the [MultiSignReserve Amendment](known-amendments.html#multisignreserve) enabled, multi-signing requires 5 XRP for the account reserve, regardless of the number of signers and signatures you use.
```

Comment:
Hmmmm.  This paragraph may need to become more complicated.  Since I have a captive audience I'll take the liberty of being long winded.  You can decide how you want to handle the actual document.

The problem here is that the section addresses two different things:

1. The cost of a transaction (the minimum fee) and
2. The reserve on an account.

The MultiSignReserve amendment only affects the reserve.  The increased minimum fee associated with multisigning a transaction is unaffected.  It still costs more to submit a successful multisigned transaction, even with the MultiSignReserve amendment.

 So, assuming you want to talk about both the reserve and the transaction cost, I'm thinking you'll need two paragraphs.  But you should write it as you see fit.